isis/python_bindings/apps/findFeaturesSegment/findFeaturesSegment.py

```diff
@@ -134,7 +134,7 @@ def segment(img_path : Path, nlines : int = MAX_LEN):
 
 def generate_cnet(params, images):
 
-    match_segment_n = images["match"]["Segment"]
+    match_segment_n = images["match"][0]["Segment"]
```

Comment:
Is this needed every time? Or only sometimes? I find it odd that it seemed to be working in some cases. Sometimes you have to check both cases.

Reply:
Hm very true, I'll have to add some sort of check then. Good catch.